.github/workflows/release.yaml

```
@@ -595,3 +595,52 @@ jobs:
         working-directory: packages/convex
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_SECRET }}
+
+  release-bun:
+    name: Release @dodopayments/bun
+    runs-on: ubuntu-latest
+    needs: release-core
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)